data/core.yaml

```
@@ -788,6 +788,10 @@ en:
       inch: in
     max_length_reached: "This string is longer than the maximum length of {maxChars} characters. Anything exceeding that length will be truncated."
     set_today: "Sets the value to today."
+    set_photo_from_viewer: "Store this photo on the currently selected map object"
+    set_photo_from_field: "Use the currently displayed photo"
+    show_photo_from_field: "Open image in viewer"
```

```suggestion
    show_photo_from_field: "Open image in street-level photo viewer"
```

Just to make it a bit more precise.